app/services/settings.js

```diff
@@ -83,6 +84,12 @@ export default Ember.Service.extend({
     return !this.readBoolean("stock.allow_box_pallet_item_addition");
   }),
 
+  /**
+   * @property {Computed<string>} paramsNotCopied lists params not to be copied during duplicate items
+   */
+  paramsNotCopied: Ember.computed(function() {
+    return this.readString("stock.params_not_to_be_copied");
```

Comment:
You can probably add a `readArray` or `readList` method, that way it's reusable and you can remove the splitting code from the controller

Reply:
Thanks ..will do that